spec/models/queue_item_spec.rb

```diff
@@ -26,6 +28,33 @@
     end
   end
 
+  describe '#rating=' do
+    let(:user) { Fabricate(:user) }
+    let(:video) { Fabricate(:video) }
+    let(:queue_item) { Fabricate(:queue_item, user: user, video: video) }
+    it 'has valid fabricators' do
```

Comment:
This is a strange spec for me. The other specs will tell you when the fabricator is producing invalid data.